Changelog.md

```diff
@@ -1,5 +1,9 @@
 # Spectrometer Changelog
 
+## Unreleased
```

Comment:
Let's have a dedicated release since, we are adding new executable in distribution